src/js/components/FormField/doc.js

```diff
@@ -42,6 +42,10 @@ export const doc = FormField => {
     pad: PropTypes.bool.description(
       'Whether to add padding to align with the padding of TextInput.',
     ),
+    placeholder: PropTypes.oneOfType([
```

Comment:
The `placeholder` shouldn't be a prop of FormField. it's a global attribute of HTML input. it shouldn't be placed on the FormField documentation.